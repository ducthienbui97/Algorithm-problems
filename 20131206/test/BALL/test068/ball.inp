8 624866 966952 771292 678744
880697 451767 232276 881951 894545 576018 553254 92185 508109 14319 795499 23633 988900 153285 436952 415157 131331 581084 338131 368925 216425 748530 306796 30559 545534 579931 792368 82179 291436 530287 263880 102246 468508 796487 190316 126891 263757 110202 548348 493957 220333 653165 978666 513741 696534 613132 856792 265860 666172 1572 869273 54973 68313 257498 358000 24318 288991 628134 308497 277883 168097 331647 456978 958851 
278434 46283 464702 271778 997303 169909 865203 323597 773359 219035 596121 192141 600031 105422 859957 503354 270554 735316 40017 738874 305292 275905 561043 152507 79751 241796 579098 849194 89028 766388 585419 481480 980206 475187 343272 297723 336350 964558 634879 135678 739370 142516 815695 732402 773581 392840 793278 211984 507277 933114 964842 868646 497550 709541 451567 346934 371519 880519 543998 227036 997728 700623 142273 879912 
532068 174765 404972 181623 570784 626318 111731 48809 947445 95252 377142 615513 481344 244429 215523 355590 238328 962175 530080 207680 685432 718116 281657 210627 262921 790092 325196 968718 315533 559012 68096 23694 236826 644083 142454 723961 555222 329059 339216 373499 371409 358204 154640 505698 605123 225997 288419 944384 395419 743361 19495 602723 814371 870486 929689 469837 862989 943850 586536 622755 891216 391996 188938 784202 
707799 750714 558679 705290 286601 779038 97883 914931 635498 494656 631744 988938 477776 376240 719344 542888 431983 592389 8597 120216 85630 377151 418409 476233 388893 504474 667587 655575 127403 236043 953586 630999 139164 708556 773590 111397 103991 593413 923681 22907 960758 787137 52818 564014 283802 203077 671457 135654 859182 961775 236321 336167 418626 663067 636837 787266 608867 145458 158030 689351 844531 509429 953560 901670 459975 253989 22060 943887 998346 948639 438624 722421 179647 313989 831649 768638 511155 146186 277424 234725 612701 289325 488253 415810 820564 214446 742676 187373 419453 564067 489095 626517 747759 301038 596229 699588 556149 530765 581804 851818 514893 682327 311735 717905 555979 961032 949631 776489 385690 560569 147643 408685 667400 619771 11533 630241 938874 905772 600029 907999 596183 553048 11113 713801 600881 358685 947758 200415 358617 94789 640808 8813 890838 716878 561319 544709 687685 634975 974312 132417 850214 876263 348877 273043 555097 777652 525052 177900 73157 796314 585308 830516 805204 987409 399856 895552 670686 843334 241518 776 897027 511039 713703 751002 337612 125461 959409 77226 931584 446112 407138 974213 352946 599224 818528 517144 998600 450819 749996 34683 24331 734209 272602 241844 653885 244543 684847 296141 589428 430674 798314 997312 399260 561823 843635 907275 573160 318521 90425 944097 478676 7813 765330 113020 89075 594829 121663 67010 964601 805423 775388 522537 776284 776728 976653 820786 329978 629 503726 383269 226495 297556 886722 105869 177034 597179 735183 953872 385066 299080 824684 407676 314175 910627 955749 10948 863748 367485 199214 17592 763092 365817 639458 500901 452147 99583 377770 34337 689103 284132 66968 290135 553143 884828 78534 43051 504399 454107 901555 297707 230636 944873 562882 10816 32255 769657 70513 96640 420239 450950 730113 878165 821905 885382 178616 62409 290054 59506 788225 215156 163800 215499 664578 691576 98352 799252 618592 722689 716295 70781 411495 145512 323414 323560 698085 170543 104130 39856 704795 553418 334246 872152 551417 701834 477930 347459 164345 134635 181257 43929 371783 19627 672955 475479 525487 345626 264434 463083 919082 118548 756017 883947 982341 103220 124572 943240 669223 85088 385553 907013 454877 318381 421075 838829 659975 965496 13326 9695 58654 94460 677181 216614 9602 52967 563898 124950 788138 794670 854067 472778 593460 663285 335579 293758 65982 683301 373387 346927 487763 781677 642992 695751 219720 974421 639821 8029 561370 370632 771390 623407 973930 28442 388825 232229 913298 707487 283000 396908 811162 466050 230177 567797 600385 854475 854339 21919 219466 463809 91968 775764 669210 297527 506870 577340 487667 294864 429877 983261 852977 598484 77737 714887 442809 623598 990109 879228 195998 463197 571487 995349 325776 682104 737775 433545 441448 932004 961902 332220 188817 291415 287443 451355 24078 409643 11805 60335 727872 547047 724756 109832 919303 243202 246728 844944 972167 335787 901632 365412 754740 719653 386297 672525 786026 115065 953957 759693 353955 179066 68841 398911 959381 946394 564845 515180 247257 514047 390962 743274 915948 903437 1791 823463 51276 877435 220541 939334 288052 864423 395596 795418 674675 350280 698415 154734 149342 823754 848213 734916 434551 686830 730418 444658 103414 475368 283982 242824 182178 497350 624392 61344 679889 749811 524177 148434 882216 507289 252001 872622 920653 217529 24309 969466 854814 641086 365274 220366 910845 122462 937774 323438 681268 475674 