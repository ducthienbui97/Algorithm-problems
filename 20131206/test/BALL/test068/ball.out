18196894002926
